packages/widget/src/pages/SettingsPage/SlippageSettings/SlippageSettings.style.tsx

```diff
@@ -0,0 +1,93 @@
+import { alpha, styled } from '@mui/material/styles';
+import { Box, ButtonBase, InputBase } from '@mui/material';
+import type { Theme } from '@mui/material';
+import { inputBaseClasses } from '@mui/material/InputBase';
+import { getCardFieldsetBackgroundColor } from '../../../utils';
+
+export const SettingsFieldSet = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  backgroundColor: getCardFieldsetBackgroundColor(theme),
+  borderRadius: Math.max(
+    theme.shape.borderRadius,
+    theme.shape.borderRadiusSecondary,
+  ),
+  padding: theme.spacing(0.5),
+  gap: theme.spacing(0.5),
+  height: '3.5rem',
+}));
+
+const slippageControlSelected = (theme: Theme) => ({
+  backgroundColor:
+    theme.palette.mode === 'dark'
+      ? theme.palette.background.default
+      : theme.palette.common.white,
+  borderRadius: theme.shape.borderRadiusSecondary,
+  boxShadow: `0px 2px 4px ${alpha(theme.palette.common.black, 0.04)}`,
+});
+
+interface SlippageDefaultProps {
+  selected?: boolean;
+}
+export const SlippageDefaultButton = styled(ButtonBase)<SlippageDefaultProps>(({
+  theme,
+  selected,
```

Comment:
I checked the use of the `selected` prop. I think because it is a boolean it doesn't get added to the DOM and so shouldForwardProp isn't needed. Same for the Input below